includes/polling/os/lenovoemc.inc.php

```diff
@@ -0,0 +1,5 @@
+<?php
+$hardware = snmp_get($device, 'deviceDescr.0', '-Ovq', 'IOMEGANAS-MIB');
+$version = 'v1';
```

Comment:
Because it doesn't seem to provide a version number anywhere (stepped through all the SNMP info), and I figured this was required. I can remove it, if that makes sense to you - no issue here!